server/backends/github/github.go

```diff
@@ -107,6 +109,12 @@ func (c *GithubClient) Link(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	redirectURL := r.FormValue("redirect_uri")
```

Comment:
nit: replace `r.FormValue("redirect_uri")` on line 125 below with `redirectURL`, to minimize the possibility of accidentally using an unvalidated value in case this code gets shifted around in the future

Reply:
Done.